README.rst

```diff
@@ -1,6 +1,8 @@
-Python pcap module
```

Comment:
Now that this is in .rst format we may want to also read it and import it for display on pypi.